docs/absolute.md

```diff
@@ -308,10 +308,13 @@ A difference of two hours will become 7200 seconds when `largestUnit` is `"secon
 However, a difference of 30 seconds will still be 30 seconds even if `largestUnit` is `"hours"`.
 
 By default, the largest unit in the result is seconds.
-Unlike other Temporal types, months and years are not allowed.
-This is because in the ISO calendar, months and years can be different lengths depending on which month is meant and whether the year is a leap year.
+Weeks, months and years are not allowed, unlike the difference methods of the other Temporal types.
+This is because months and years can be different lengths depending on the calendar in use, which month is meant, and whether the year is a leap year.
```

Comment:
Most importantly, the length of a month depends on the start date, and you don't know the start date of an Absolute unless you know the time zone.  The calendar system is secondary.